packages/contact-service/src/lib/tests/contact-service.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { ContactService } from '../ContactService';
+import { Contact } from '../../model/Contact';
+import { eventBus } from '@adorsys-gis/event-bus';
+
+describe('ContactService', () => {
+  let contactService: ContactService;
+
+  beforeEach(() => {
+    // Initialize the ContactService before each test
+    contactService = new ContactService(eventBus);
+  });
+
+  afterEach(async () => {
+    // Clear all contacts after each test
+    const allContacts = await contactService.getAllContacts();
+    for (const contact of allContacts) {
+      await contactService.deleteContact(contact.id!);
+    }
+  });
+
+  it('should create a new contact', async () => {
+    const newContact: Contact = {
+      id: 1,
+      name: 'John Doe',
+      did: 'did:example:123456',
+    };
+
+    await contactService.createContact(newContact);
+    const retrievedContact = await contactService.getContact(newContact.id!);
+
+    expect(retrievedContact).toEqual(
+      expect.objectContaining({
+        name: newContact.name,
+        did: newContact.did,
+      }),
+    );
+  });
+
+  it('should retrieve a contact by ID', async () => {
+    const contact: Contact = {
+      id: 2,
+      name: 'Charlie Brown',
+      did: 'did:example:789012',
+    };
+
+    await contactService.createContact(contact);
+
+    const retrievedContact = await contactService.getContact(2);
```

Comment:
A more realistic usage will not have an ID in the contact to create, so prefer not including one. The createContact method should now return the contact created with a generated ID, which in turn you can use for the getContact step (makes sense in a test).